downstream/modules/troubleshooting-aap/proc-troubleshoot-aap-packages.adoc

```diff
@@ -0,0 +1,6 @@
+[id="troubleshoot-aap-packages"]
+= Issue - Cannot locate certain packages that come bundled with the {PlatformNameShort} installer
```

Comment:
This is up to you, but at some point you might want to make sure this troubleshooting guide is structured consistently with our other troubleshooting documents we've published. For example, the[ Troubleshooting section in the Automation controller user guide](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/automation_controller_administration_guide/controller-troubleshooting), or the [Lightspeed troubleshooting section](https://access.redhat.com/documentation/en-us/red_hat_ansible_lightspeed_with_ibm_watsonx_code_assistant/2.x_latest/html/red_hat_ansible_lightspeed_with_ibm_watsonx_code_assistant_user_guide/troubleshooting-lightspeed_lightspeed-user-guide) (which I realize is not technically part of the ansible doc set). I'm thinking primarily of the Issue -- issue description title formatting.

Reply:
Great point Hala! Lorena mentioned this during her editorial review. She reached out to Emily Murphy on the modular docs team and they discussed the formatting of this doc. 

Here's some feedback Lorena shared: "_We like the use of 'Issue' at the beginning of each title and that the title focuses on the problem that users are having. In that sense, it is different from what we see in regular procedures in our documentation where we focus on what users are doing to solve a problem or achieve a goal._"